NanoVNASaver/RFTools.py

```diff
@@ -1,4 +1,6 @@
-#  NanoVNASaver - a python program to view and export Touchstone data from a NanoVNA
+#  NanoVNASaver - a python program to view and export Touchstone data
+#  from a NanoVNA
+#
```

Comment:
Max line length is 120 in this project, as per PyCharm defaults. I can fix this on my end, though.